source/read-write-concern/read-write-concern.rst

```diff
@@ -313,6 +313,33 @@ With MaxWireVersion < 4, ``writeConcern`` MUST be omitted when sending ``findAnd
 .. note ::
     Driver documentation SHOULD include a warning in their server 3.2 compatible releases that an elevated ``WriteConcern`` may cause performance degradation when using ``findAndModify``. This is because ``findAndModify`` will now be honoring a potentially high latency setting where it did not before.
 
+Helpers for other commands that write
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If your driver has helpers for commands that write other than those discussed above, they SHOULD accept a ``writeConcern`` parameter.
```

Comment:
I'm not sure if you meant to take out the list or to just not mention helpers. Let me know if my updates correspond to what you were suggesting. Thanks!